llvm/lib/Target/Xtensa/XtensaRegisterInfo.cpp

```diff
@@ -57,10 +58,47 @@ BitVector XtensaRegisterInfo::getReservedRegs(const MachineFunction &MF) const {
   return Reserved;
 }
 
+bool XtensaRegisterInfo::eliminateFI(MachineBasicBlock::iterator II,
```

Comment:
Why don't just inline it?

Reply:
I merged functions and removed eliminateFI.